x-pack/test/cases_api_integration/security_and_spaces/tests/common/user_actions/migrations.ts

```diff
@@ -23,6 +23,234 @@ export default function createGetTests({ getService }: FtrProviderContext) {
   const es = getService('es');
 
   describe('migrations', () => {
+    describe('7.7.1', () => {
+      const CASE_ID = '2ea28c10-7855-11eb-9ca6-83ec5acb735f';
+
+      before(async () => {
+        await esArchiver.load('x-pack/test/functional/es_archives/cases/migrations/7.7.1');
```

Comment:
Do you think we could convert this to a kbn archiver format and import it instead? My concern is we've had flakyness around es archiver in the past and we should probably try not to add more tests that use it. We'll have to strip out the saved objects that aren't importable though (configuration, config mapping)